README.md

```diff
@@ -4,8 +4,20 @@
 
 # Jaeger - a Distributed Tracing System
 
-  * See [documentation](http://jaeger.readthedocs.io/en/latest/) for getting started, operational details, and other information.
-  * Read a blog post [Evolving Distributed Tracing at Uber](https://eng.uber.com/distributed-tracing/).
+Jaeger, inspired by [Dapper][dapper] and [OpenZipkin](http://zipkin.io),
+is a distributed tracing system released as open source by [Uber Technologies][ubeross].
```

Comment:
I don't think it's redundant because "system by Uber" reads quite differently and ignores later contributions by other people.